lib/widgets/subscription_list.dart

```diff
@@ -200,15 +204,18 @@ class SubscriptionItem extends StatelessWidget {
     super.key,
     required this.subscription,
     required this.unreadCount,
+    required this.hasUnmutedUnreads,
```

Comment:
This field is redundant with `unreadCount`. The code in this widget's build method, and the code at the call site constructing one of these widgets, will both be clearer if there's just the one field — otherwise the reader has to wonder about the case where `hasUnmutedUnreads` doesn't agree with `unreadCount > 0`.